mlflow/spark.py

```diff
@@ -660,6 +673,34 @@ def _load_model(model_uri, dfs_tmpdir_base=None):
     return PipelineModel.load(model_uri)
 
 
+def _log_model_via_mlflowdbfs(artifact_repo, artifact_path, model):
+    mlflowdbfs_path = databricks_utils.get_mlflowdbfs_path(artifact_repo.run_id, artifact_path)
+    databricks_utils.set_databricks_host_creds_to_credential_context(
```

Comment:
Nit: `set_databricks_host_creds_to_credential_context` phrasing is a bit awkward. Let's rename this method. Perhaps `add_databricks_host_creds_to_credential_context`.